### PR TITLE
feat(incorrect-information-button): Information button about how to change your data

### PR DESCRIPTION
**Pull request by Zeroks77**
Here we go again, :D 

I have no idea why my workspace formats the files differently, I have started to change
some back again....

### CODE REVIEW

**Comment by mschwrdtnr** [comment]
I think this https://github.com/T-Systems-MMS/phonebook/issues/174#issuecomment-540961624 is for you @Zeroks77

File: Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.scss [hunk 1]
```diff
@@ -1,6 +1,27 @@
 @import 'media-queries';
 @import '~@angular/material/theming';
 
+@keyframes changeColor {
+  0% {
+    color: #ff0000;
```
Comment (DanielHabenicht) [inline Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.scss]: That's got nothing to do with the wrong information button?

**Comment by DanielHabenicht** [comment]
Thanks for the new PR, There are still some issues: 
 - translate the new text so it is available in german too
 - the position of the button is suboptimal as it takes a whole column, equally to one information column but does not provide any information and brings only marginal functionality to the user especially at its current state. I would prefer it next to the "share" and "fav" buttons. 
 - The button is displayed on every user profile and says "Change user information". This seems counter-intuitive as you should not be able to change an other users profile. How about "Report incorrect data"
 - I am not convinced that we really need a whole dialog for the information displayed, how about a simple snackbar? Please discuss with @mschwrdtnr and @paule96 (I will add more for the code review as you decide) 
 - I would drop the formality in the popup and get straight to the point 

If possible please fix the format issues by checking in the according vscode settings. (might be line length and space handling)

File: Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.scss (hunk 1 above)
Comment (Zeroks77) [reply Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.scss]: my b 
I played with the css and forget to undo befor stage my changes

**Comment by Zeroks77** [comment]
@DanielHabenicht  what about just show the button when u r on ur profile ? 
like the profil picutre change button 

 Besides are u sure u want text there ? it looks pretty dumb, text next to the beutiful icons

**Comment by mschwrdtnr** [comment]
@all-contributors please add @Zeroks77 for code

**Comment by DanielHabenicht** [comment]
Everything you can do at the moment to fix wrong information is to write an email. So there is no downside displaying the button on every user profile. 

Well, but it doesn't take a whole column there. Maybe you can find another way? Make it a little bit smaller or take a look at the other internal phonebook.

**Comment by Zeroks77** [comment]
@DanielHabenicht  look at it i guess @paule96 und me found a greate solution :)

**Comment by Zeroks77** [comment]
I will create a new PR with my changes.

@mschwrdtnr and @DanielHabenicht could you provide a .pretierrc where you define what settings you want in your project.

Since this Pr wanna change binary Files, i think we should stop it. Please dont close it untile #350 is solved. Elsewhise i gonna forget it :)

**Comment by DanielHabenicht** [comment]
Ok I will wait till then just ping me in the new PR

**Pull request closed without merging**